src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,18 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveItem: {
+      type: ConversationItemType,
+      resolve: (parent, _args, { artworkLoader }) => {
+        if (parent.item_type === "Artwork") {
+          return artworkLoader(parent.properties.id).then((artwork) => ({
+            ...artwork,
+            __typename: "Artwork",
+          }))
+        }
+        throw new Error("Only artworks are supported.")
+      },
+    },
```

Comment:
I'm in favor of this option because: 
1. We don't need do any data massaging
2. We are encapsulating this live artwork versus convoluting it with the other items (it feels a little cleaner).

Reply:
I agree on using a new field for this use case. While we are not going to support multiple conversation items in a conversation anytime soon, it might still be a good idea to keep it consistent with the current `items` and name this in plural form.